src/test/Figure/Population.py

```diff
@@ -0,0 +1,29 @@
+from typing import Optional
+
+from converter import CamelCaseModel, DataProductDefinition
+from pydantic import Field
+
+
+class PopulationResponse(CamelCaseModel):
+    label: str = Field(..., title="Data reference label")
+    source: str = Field(..., title="Data source name")
+    value: int = Field(..., title="The population value")
```

Comment:
I'd likely call this `population` just to make it easy to see what it is just from the payload.